strategy/live_trader.py

```
@@ -337,6 +411,265 @@ def __init__(self):
+        """Cancel a single active order on Polymarket CLOB."""
+        if not order_id:
+            return False
+        client = self.get_clob_client()
+        if not client:
+            return False
+        try:
+            if hasattr(client, "cancel"):
+                try:
+                    res = client.cancel(order_id)
+                    log.info("Cancelled order %s -> %s", order_id, res)
+                    return True
```

Ruff (0.16.3)

[warning] Consider moving this statement to an `else` block

(TRY300)